tests/acceptance/acceptance/TwigEnvironment.feature

```diff
@@ -0,0 +1,36 @@
+@symfony-common
+Feature: LockableTrait
+
+    Background:
+        Given I have the following config
+      """
```

Comment:
indentations can be matched

Reply:
Fixed, I also added an `.editorconfig` file.
This way the IDE is automatically using the same indentation than the project.